src/jemalloc.c

```diff
@@ -57,6 +57,9 @@ const char	*je_malloc_conf_2_conf_harder
 #endif
     ;
 
+char malloc_conf_file[PATH_MAX + 1];
+char malloc_conf_env_var[PATH_MAX + 1];
```

Comment:
Discussed offline with Sherry about how to avoid the two static buffers here.

Reply:
updated!